devel/autoconf/Portfile

```diff
@@ -11,6 +11,7 @@ categories          devel
 # https://www.gnu.org/licenses/autoconf-exception.html
 platforms           darwin
 supported_archs     noarch
+configure.cxx_stdlib
```

Comment:
Not sure you meant to include this in this PR ... guessing it is "collateral changes".

Reply:
It was in the original PR, I did not add it.

Reply:
Was in original PR

Reply:
For ports that don't actually use a compiler, it's easier to turn off dependencies on libc++-capable compilers like this, instead of blacklisting stuff.